source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -270,7 +270,12 @@ export default class CreepVote extends React.Component {
         <div className="mb-5">
           <div className="col-12 text-center">
             <h3 className="h2-heading mb-1">
-              {this.state.totalVotes + 1} Votes — invite your friends!
+              <Localized
+                id="voted-header"
+                $voteCount={this.state.totalVotes + 1}
+              >
+                {`{$voteCount} Vote — invite your friends!`}
```

Comment:
especially this is _incredibly_ bad - template strings already use `${thing}` to get values into the string, if we also introduce `{$thing}` as valid syntax, we've just guaranteed that reviewers are going to miss errors around using one where the other should have been used.

Reply:
The string here is intended as a fallback for localization. The `{$voteCount}` is Fluent syntax right there. Technically, the string is not required, because `@fluent/react` will insert the translation in its place anyways. But it helps when grepping the code for copy, and in the future we might offer tools to autoextract translations directly from JSX. There's one such tool written by the community: https://github.com/udacity/fluent-react-utils/tree/master/packages/extraction